libnmstate/netapplier.py

```diff
@@ -243,7 +244,17 @@ def _disable_ipv6(desired_state):
 
     This is an intermediate workaround for https://bugzilla.redhat.com/1643841.
     """
+    has_disabled_iface = False
     for ifstate in six.viewvalues(desired_state.interfaces):
         ipv6_state = ifstate.get(schema.Interface.IPV6, {})
         if ipv6_state.get('enabled') is False:
+            has_disabled_iface = True
             sysctl.disable_ipv6(ifstate[schema.Interface.NAME])
+
+    if has_disabled_iface:
```

Comment:
I do not think we should have such a warning at all. The user should not be expected to know such details, we are here to abstract for him everything.
We can add to the release-notes/docs the limitations.